plutus-core/changelog.d/20230411_100505_marty.stumpf_plt_5439_ensure_global_uniqueness.md

```diff
@@ -0,0 +1,7 @@
+### Added
+
+- Unit tests for global uniqueness for the PIR inliner.
```

Comment:
I don't think we need to provide changelogs for out testsuite, generally.

Reply:
Can we have some more general rules for change logs? It feels a bit random atm.